### PR TITLE
React_add-todo-form

### PR DESCRIPTION
**Pull request by AnastasiiaHombalevska**
🦊 [DEMO LINK](https://AnastasiiaHombalevska.github.io/react_add-todo-form/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,27 +1,106 @@
...
+            value={selectedUserId}
+            onChange={handleOnChangeUser}
+            required
+          >
             <option value="0" disabled>
               Choose a user
             </option>
+
+            {usersFromServer.map(user => (
+              <option value={user.id} key={user.id}>
```
Comment (GUSILLUS) [inline src/App.tsx:97]: use destruction assignment to user objecy

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1 +1,18 @@
...
+} : { todo: TodosProps }) => (
+  <article
+    data-id={16}
+    className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed,
+    })}
+  >
+    <h2 className={classNames('TodoInfo__title')}>{todo.title}</h2>
+
+    {todo.user && <UserInfo user={todo.user} key={todo.user.id} />}
```
Comment (GUSILLUS) [inline src/components/TodoInfo/TodoInfo.tsx:16]: same with todo

File: src/components/UserInfo/UserInfo.tsx [hunk 3]
```diff
@@ -1 +1,8 @@
-export const UserInfo = () => {};
+import classNames from 'classnames';
+import { UsersProps } from '../../types/User';
+
+export const UserInfo = ({ user } : { user: UsersProps }) => (
+  <a className={classNames('UserInfo')} href={`mailto:${user.email}`}>
```
Comment (GUSILLUS) [inline src/components/UserInfo/UserInfo.tsx:5]: and here also)

**Review by GUSILLUS — CHANGES_REQUESTED** [review]
gj!

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**